docs/en/connector-v2/source/dingtalk.md

```diff
@@ -0,0 +1,76 @@
+# DingTalk
+
+> DinkTalk source connector
+
+## Description
+
+A source plugin which use DingTalk API
+
+## Key features
+
+- [x] [batch](../../concept/connector-v2-features.md)
+- [ ] [stream](../../concept/connector-v2-features.md)
+- [x] [exactly-once](../../concept/connector-v2-features.md)
```

Comment:
Can you describe how this connector implements exactly-once? I can't find the relevant content from the code. If I miss something, please let me know.